spyderlib/widgets/arraybuilder.py

```diff
@@ -202,11 +206,11 @@ def __init__(self, parent, inline=True, offset=0):
         if inline:
             self._button_help.setToolTip(self._help_inline)
             self._text = NumpyArrayInline(self)
-            self._widget = self._text
+            self.widget = self._text

```

Comment:
This attribute should stay private even if it's used in the tests. The tests are internal, it's totally fine if they depend on the implementation.
Being able to test correctly is one of the reason that there is no real private attribute in python, so that even private attributes can be tested.